blog/announcements.xml

```diff
@@ -1,6 +1,104 @@
 <?xml version="1.0" encoding="utf-8" ?>
 
 <news>
+<item>
+    <pubDate>Tue, 30 November 2022 19:00:00 UTC</pubDate>
```

Comment:
```suggestion
    <pubDate>Wed, 30 November 2022 19:00:00 UTC</pubDate>
```

Also: check time?

Reply:
The time was quite precise at the moment of opening this PR.

Reply:
Sure, but do we wanna set it to be more accurate (say, XX:30)? Or set it after the fact? Or...?

Reply:
Fixed the date, the time is maybe roughly around release time.

Reply:
Tagged at 18:42 UTC, so 19:00 UTC is fine.